### DISCUSSION TITLE
%E5%BC%80%E6%BA%90%EF%BC%81%E5%85%8D%E8%B4%B9%EF%BC%81%E6%9C%AC%E5%9C%B0%E9%83%A8%E7%BD%B2%E7%9A%84%E9%9B%B7%E6%80%BB%E5%A3%B0%E7%BA%BF%EF%BC%81/

### DISCUSSION THREAD

**Discussion by giscus** [shangy1yi/shangy1yi.github.io#7] [General]
# %E5%BC%80%E6%BA%90%EF%BC%81%E5%85%8D%E8%B4%B9%EF%BC%81%E6%9C%AC%E5%9C%B0%E9%83%A8%E7%BD%B2%E7%9A%84%E9%9B%B7%E6%80%BB%E5%A3%B0%E7%BA%BF%EF%BC%81/

转载自：https://link.syouiti.com/leijunvits 需要下载的1.整合包2.语音模型(123网盘 百度网盘)3.Python PyTorch CUDA（可忽略）自己去官网下 使用安装安装好整合包后在整合包内打开cmd输入 1pip install -r requirements.txt -i https://pypi.tuna.tsinghua.edu.cn/simpl

https://blog.syouiti.com/%E5%BC%80%E6%BA%90%EF%BC%81%E5%85%8D%E8%B4%B9%EF%BC%81%E6%9C%AC%E5%9C%B0%E9%83%A8%E7%BD%B2%E7%9A%84%E9%9B%B7%E6%80%BB%E5%A3%B0%E7%BA%BF%EF%BC%81/

**Comment by godsoul** [comment]
语音模型链接失效了，能补下吗
